_posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: The Road To Publish My Short ebook
+date: 2019-10-21 20:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Writing]
+image:
+image2:
+author: Valentino Urbano
+---
+
+# The Beginning
+
+I'm turning my "Migrating From WordPress To Jekyll" blog post into a small ebook. I will be revisiting each entry and edit it for the medium.
+
+Firstly I had to decide how to publish it. I decided to release it for free online, while at the same time release a pdf and epub version on both Amazon and Gumroad for a small fee (still not decided). I don't expect to get any sale, but it is going to be an interesting process to go through nonetheless both generating and formatting the book as well as figuring out how to set up a page on both Amazon and Gumroad.
```

Comment:
Change word order - > online for free

release - > releasing? 

sale - > sales